var/spack/repos/builtin/packages/py-keyring/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyKeyring(PythonPackage):
+    """The Python keyring library provides an easy way to access the system keyring
+    service from python. It can be used in any application that needs safe password
+    storage."""
+
+    homepage = "https://github.com/jaraco/keyring"
+    pypi     = "keyring/keyring-23.0.1.tar.gz"
+
```

Comment:
Are you willing to be the package maintainer?

Reply:
Nvm.  I vaguely recall I've asked you this before. :)

Reply:
I'm technically the maintainer for all Python packages